src/playlist-loader.js

```diff
@@ -259,9 +259,9 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
       if (playlist && playlist.attributes) {
         bandwidth = playlist.attributes.BANDWIDTH;
       }
-      return bandwidth <= currentBandwidth;
+      return bandwidth < currentBandwidth;
 
-    }).length > 1);
+    }).length >= 1);
```

Comment:
I think this boolean logic could use some cleaning up. Right now it says `length not greater than or equal to one`  which is difficult to wrap my brain around. I think a better one is simply `length is zero`.